onboarding/mentors.md

```diff
@@ -31,53 +71,83 @@ It's up to the mentoring pair to find what works best for them and to ensure the
+  If so, check in on where they are and offer to be a resource on things there. Some tasks might stand out as good
+  things to do in between People Ops meetings, while working nearby each other or to pair on.
+
+**Your first week**
 
-### Your first week
-- Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira, github, notion, heroku, aws+hokusai, zeplin, ...)
+- Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale
+  steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira,
+  github, notion, heroku, aws+hokusai, zeplin, ...)
 - Look for chances to pair or work alongside one another throughout the week.
-  - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just setting it up, breaking and fixing a test.
+  - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just
+    setting it up, breaking and fixing a test.
   - If they haven't set up hokusai yet, consider pairing on that.
-- Try to have another meeting towards the end of the week- **especially** if they are leaving New York. Revisit things you talked about in the first meeting and look for areas that could have gone better. Consider amending relevant documents like this one.
+- Try to have another meeting towards the end of the week- **especially** if they are leaving New York. Revisit
+  things you talked about in the first meeting and look for areas that could have gone better. Consider amending
+  relevant documents like this one.
 
-### Your first year
-- New hires usually begin with a sprint-by-sprint rotation through our teams, a whirlwind that can last upwards of 2 months. It's good to check in with your mentee during this time.
+**Your first year**
+
+- New hires usually begin with a sprint-by-sprint rotation through our teams, a whirlwind that can last upwards of
+  2 months. It's good to check in with your mentee during this time.
 - Consider reaching out to other mentors as well
 - Periodically check in with yourself on what you think you know about your mentee:
-  - How are they feeling at work? It's good to be aware (if you can) of comfort level, new technologies and more dynamic pieces of how they are getting situated at Artsy.
-  - Are there any opportunities to pair together (or for them to pair with someone else) on a new technology or domain area?
-- Many mentoring pairs continue scheduling semi-weekly coffee chats in the weeks and months going forward. As the relationship becomes less formal these can be great to keep in touch with a team member you have a special relationship with!
+  - How are they feeling at work? It's good to be aware (if you can) of comfort level, new technologies and more
```

Comment:
Rotations tend to introduce new product areas and systems, and those are natural opportunities for a mentor to provide a code tour, product demo, or informal history/background.